avm/res/container-registry/registry/main.bicep

```diff
@@ -290,6 +293,16 @@ resource registry 'Microsoft.ContainerRegistry/registries@2023-06-01-preview' =
       : null
     zoneRedundancy: acrSku == 'Premium' ? zoneRedundancy : null
   }
+
+  resource registry_scopeMap 'scopeMaps' = [
```

Comment:
In general I am happy with the implementation, but this is a subresource and should be written in a submodule like all AVM modules.